src/settings/model.ts

```
@@ -126,10 +168,14 @@ export function sanitizeSettings(settings: CopilotSettings): CopilotSettings {
 
   // Stuff in settings are string even when the interface has number type!
   const temperature = Number(settingsToSanitize.temperature);
-  sanitizedSettings.temperature = isNaN(temperature) ? DEFAULT_SETTINGS.temperature : temperature;
+  sanitizedSettings.temperature = isNaN(temperature)
+    ? DEFAULT_SETTINGS.temperature
+    : temperature;
 
   const maxTokens = Number(settingsToSanitize.maxTokens);
-  sanitizedSettings.maxTokens = isNaN(maxTokens) ? DEFAULT_SETTINGS.maxTokens : maxTokens;
+  sanitizedSettings.maxTokens = isNaN(maxTokens)
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)